Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs

```diff
@@ -12,3 +13,47 @@ let ( / ) a b =
     ((parsed a) / (parsed b)).InnerSimplified
 let ( ** ) a b =
     ((parsed a).Pow(parsed b)).InnerSimplified
+let ( ~~~ ) a = 
+    (negation (parsed a)).InnerSimplified
+let ( ||| ) a b = 
+    (union (parsed a) (parsed b)).InnerSimplified
+let ( ^^^ ) a b = 
+    (intersect (parsed a) (parsed b)).InnerSimplified
```

Comment:
I think... nobody's gonna use those...

Reply:
Intersection should be `&&&`. `^^^` is exclusive or.

Reply:
I'm not sure we need a separate operator for it, to be honest. A function is fine imo, this operator would definitely be not as common as >, <, +, -, etc.